### ISSUE TITLE
[Bug]: Shift + Home doesn't select all text in TagPicker

### ISSUE THREAD

**Issue by joannakuo** [microsoft/fluentui#34888]
### Component

TagPicker

### Package version

^9.64.0

### React version

17.0.1

### Environment

```shell
System:
    OS: Linux 6.8 Ubuntu 22.04.5 LTS 22.04.5 LTS (Jammy Jellyfish)
    CPU: (32) x64 AMD EPYC 7763 64-Core Processor
    Memory: 118.53 GB / 125.79 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Browsers:
    Chrome: 136.0.7103.59
```

### Current Behavior

Shift + home doesn't select all text in tag picker, but it does in Fluent 8 peoplePicker component.

### Expected Behavior

The expected behavior is pressing shift + home will select all text in the TagPicker.

### Reproduction

https://react.fluentui.dev/?path=/docs/components-tagpicker--docs

### Steps to reproduce

1. Go to https://react.fluentui.dev/?path=/docs/components-tagpicker--docs
2. In the default TagPicker, type in text
3. Press shift + home, observe the inputted text doesn't get highlighted.

### Are you reporting an Accessibility issue?

None

### Suggested severity

Medium - Has workaround

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by Hotell** [comment]
confirming bug. instead of selecting text it focuses list box item

https://github.com/user-attachments/assets/792fc7e8-0db3-4c4b-a97d-7158fe6cb5d7

**Comment by joannakuo** [comment]
I was able to repro on my Windows machine in Fluent 8 people picker. Pressing Shift + Home highlighted all the text in the input. If the cursor is at the start of the text pressing Shift + End should also highlight the text. When I test in Fluent 9, I see Shift + Home goes to the start of the list, and Shift + End goes to the end. I don't think this bug is accessibility related, but a bug related to keyboard shortcuts supported by Windows. Based on the support documents below, I think the TagPicker input should support the keyboard shortcuts related to selecting text.

Best,
Joanna

https://support.microsoft.com/en-us/windows/keyboard-shortcuts-in-windows-dcc61a57-8ff0-cffe-9796-cb9706c75eec
***@***.***

**Comment by mainframev** [comment]
> confirming bug. instead of selecting text it focuses list box item
> 
>  v9_dropdown_select.bug.mov

I guess this is not right equivalent sequence on MacOS 🤔 On MacOS it works fine with `"⇧ + ⌘ + ←" / "⇧ + ⌘ + →"` (shift+cmd+left / right arrow), so bug should be only Windows related
